release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -1636,65 +1090,25 @@ references the correct API and works with the web console UI.
-create a new image stream for internal image selection; it already exists.
-(link:https://bugzilla.redhat.com/show_bug.cgiid=1808280[*BZ#1808280*]
-
-* In a Kubernetes deployment, if the images from the external image registry had
-tags such as `openshift/hello-world:1.0`, then the tags were not being applied.
-The user was unable to import external images with tags. With this big fix, the
-proper tag for the deployment is now passed.
-(link:https://bugzilla.redhat.com/show_bug.cgiid=1801736[*BZ#1801736*]
-
-* Previously, when two consecutive rollouts failed, the *Topology* view showed
-failed pods instead of displaying the last active revision. With this bug fix,
-when a rollout fails, the last active revision is displayed.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1760828[*BZ#1760828*])
-
-* Previously, existing imagestreams in a namespace were not detected when
-creating an application. This occurred when users with limited cluster-wide
-permissions used the *Container Image* -> *Image* name from the internal
-registry* options in the *Add* page. Now, the imagestream fetching logic has
-been moved from the cluster level to the namespace level, which enables a user
-with permission to a namespace to see imagestreams in that namespace.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1784264[*BZ#1784264*])
-
-* Knative service and revision resources did not have binding or visual
-connectors in the Topology view and therefore Knative workloads could not
-connect to other workloads. These resources now have connectors in the Topology
-view and can connect to other workloads.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1779201[*BZ#1779201*])
-
-* When the Eclipse Che Operator was installed and configured, the Topology view
-showed the Git icon instead of the Che icon. This provided no indication to the
-user that they could click the icon to access the Che workspace. The Topology
-view now correctly displays the Che icon when Che is configured, making it
-easier for the user to access the Che workspace.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1780338[*BZ#1780338*])
-
-* Creating a Knative service using the CLI while the Topology view was open
-caused a GUI error. Checks have been added to handle this workload without
-causing a GUI error.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1781188[*BZ#1781188*])
+* Serverless resources like the `Service` YAML file are listed as `v1beta1` instead of `v1`. However, `v1beta1` is deprecated. With this bug fix, the `apiVersion` is updated to `v1`. (link:https://bugzilla.redhat.com/show_bug.cgiid=1796421[*BZ#1796421*])
+
+* A service binding request (SBR) in the topology is associated with *Revisions* in the *Topology* view. Therefore, no new revision will get the associated secrets. The SBR should go through the knative service; secrets would be injected and new revisions would be created. (link:https://bugzilla.redhat.com/show_bug.cgiid=1798288[*BZ#1798288*]
```

Comment:
I'm guessing a lot of these lowercase/no-markup instances of "knative" could probably be uppercased "Knative".